.github/workflows/builds.yml

```diff
@@ -1348,6 +1348,87 @@ jobs:
+    name: AlmaLinux 9 (Minimal/Recommended Build)
+    runs-on: ubuntu-latest
+    container: almalinux:9
+    needs: [prepare-deps]
+    steps:
+      # Cache Rust stuff.
+      - name: Cache cargo registry
+        uses: actions/cache@13aacd865c20de90d75de3b17ebe84f7a17d57d2
+        with:
+          path: ~/.cargo
+          key: ${{ github.job }}-cargo
+
+      - name: Cache RPMs
+        uses: actions/cache@13aacd865c20de90d75de3b17ebe84f7a17d57d2
+        with:
+          path: /var/cache/dnf
+          key: ${{ github.job }}-dnf
+      - run: echo "keepcache=1" >> /etc/dnf/dnf.conf
+
+      - name: Determine number of CPUs
+        run: echo CPUS=$(nproc --all) >> $GITHUB_ENV
+
+      - name: Enable required repositories
+        run: |
+          dnf -y update
+          dnf -y install dnf-plugins-core
+          dnf config-manager --set-enable crb
+
+      - name: Install git dependencies
+        run: |
+          dnf -y install \
+            sudo \
+            git \
+            libtool \
+            which
+
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+      - run: git config --global --add safe.directory /__w/suricata/suricata
+      - uses: actions/download-artifact@c850b930e6ba138125429b7e5c93fc707a7f8427
+        with:
+          name: prep
+          path: prep
+      - run: tar xf prep/libhtp.tar.gz
+      - run: tar xf prep/suricata-update.tar.gz
+      - run: tar xf prep/suricata-verify.tar.gz
+      - run: ./autogen.sh
+
+      - name: Install minimal dependencies
+        run: |
+          # install-guide-documentation tag start: Minimal RPM-based dependencies
+          sudo dnf install -y rustc cargo
+          cargo install --force cbindgen
+          echo 'export PATH+=":$HOME/.cargo/bin"' >> ~/.bashrc && export PATH+=":$HOME/.cargo/bin"
+          sudo dnf install -y gcc gcc-c++ jansson-devel libpcap-devel \
+            libyaml-devel make pcre2-devel zlib-devel
+          # install-guide-documentation tag end: Minimal RPM-based dependencies
+          echo "$HOME/.cargo/bin" >> $GITHUB_PATH
+
+      - run: CFLAGS="${DEFAULT_CFLAGS}" ./configure
+      - run: make -j ${{ env.CPUS }}
+      - run: ./src/suricata --build-info # check if we can run Suricata
+
+      # Recommended dependencies
+      - name: Install recommended dependencies
+        run: |
+          # install-guide-documentation tag start: Recommended RPM-based dependencies
+          sudo dnf install -y epel-release
+          sudo dnf install -y autoconf automake clang diffutils dpdk-devel \
+            file-devel hiredis-devel hyperscan-devel jansson-devel libbpf-devel \
+            libcap-ng-devel libevent-devel libmaxminddb-devel libnet-devel \
+            libnetfilter_queue-devel libnfnetlink-devel libpcap-devel \
+            libyaml-devel llvm-toolset lua-devel lz4-devel make \
+            nspr-devel nss-devel pcre2-devel pkgconfig python3-devel \
+            python3-sphinx python3-yaml zlib-devel
```

Comment:
I'd remove clang, hiredis-devel, libeven-devel, libmaxminddb-devel, is llvm-toolset needed? nspr-devel, nss-devel can go as well.  Python3-sphinx shouldn't be there for end-user dependencies.